.travis.yml

```diff
@@ -25,10 +25,15 @@ before_install:
   - conda update --yes conda
 
 install:
-  - conda install --yes python=$TRAVIS_PYTHON_VERSION pip numpy scipy pytest
+  - conda install --yes python=$TRAVIS_PYTHON_VERSION pip numpy scipy pytest matplotlib Pillow
   - pip install pep8 python-coveralls pytest-cov
   - pip install -e .
 
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
```

Comment:
You could have just used xvfb-run wrapper around py.test call... But this one is as fine I guess